src/device/wg_controller.rs

```diff
@@ -4,16 +4,17 @@ use std::collections::{BTreeMap, HashMap, HashSet};
 use std::iter::FromIterator;
 use std::net::IpAddr;
 use std::sync::Arc;
-use std::time::Duration;
+use std::time::{Duration, Instant};
```

Comment:
It's better to use tokio::time::Instant, it reacts to tokio mock'ed time (pause, unpause, advance etc) and is aready used in most of telio codebase

Reply:
I guess it comes from previous code, but probably would be nice to switch :sweat_smile: